src/utils/Util.js

```diff
@@ -8,6 +8,7 @@ import remote from 'remote';
 var app = remote.require('app');
 
 module.exports = {
+  folders: [],
```

Comment:
Yeah, the folder attribute is where you stored the array with the mounted folders in the VM. You use this attribute in rc/components/ContainerSettingsVolumes.react.js line 20, and src/utils/VirtualBoxUtil.js line 76.